app/src/main/java/org/simple/clinic/navigation/v2/History.kt

```diff
@@ -0,0 +1,33 @@
+
+package org.simple.clinic.navigation.v2
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class History(val requests: List<NavRequest>) : Parcelable {
```

Comment:
We could do that, but we don't gain anything by it. The current list API lets me use standard extension functions like `dropLast` and supports immutability. Moving to use the `ArrayDeque` means that we cannot use those APIs and this become a little less easy to manage because they are not stateful.

I'd rather leave it as `List` for now and then maybe move to `ArrayDeque` if we find that it is a better fit later on.